scripts/modules/form.js

```diff
@@ -0,0 +1,3 @@
+export function formUser() {
```

Comment:
I don't see how formUser is used. Could we delete it?

Reply:
Fixed